packages/prisma/schema.prisma

```diff
@@ -366,6 +372,7 @@ model User {
   updatedTranslations        EventTypeTranslation[]       @relation("UpdatedEventTypeTranslations")
   createdWatchlists          Watchlist[]                  @relation("CreatedWatchlists")
   updatedWatchlists          Watchlist[]                  @relation("UpdatedWatchlists")
+  creationSource             CreationSource?
```

Comment:
We don't provide a default here to ensure we have all our bases covered. Any null value here for new users should tell us that we might have missed a route.